src/seedu/addressbook/commands/Command.java

```diff
@@ -27,6 +28,13 @@ protected Command() {
     }
 
     /**
+     * Returns true for mutated data.
+     */
+     public boolean isMutating() {
+         return mutation;
```

Comment:
As we discussed in the tutorial, it's not a recommended to have a default implementation of `isMutating`, as it's not clear whether Command is by default mutating or not (unlike the example on `adjustSalary`).

Since Command is an abstract class, why not make this method abstract?